p2p/agent.go

```diff
@@ -191,8 +207,8 @@ func (p *Agent) Stop(ctx context.Context) error {
 	return nil
 }
 
-// Broadcast sends a broadcast message to the whole network
-func (p *Agent) Broadcast(ctx context.Context, msg proto.Message) (err error) {
+// BroadcastOutbound sends a broadcast message to the whole network
+func (p *Agent) BroadcastOutbound(ctx context.Context, msg proto.Message) (err error) {
```

Comment:
If you want to distinguish the inbound/outbound explicitly. What if call this api `SendBroadcastMessage`, and the handler `OnReceivingBroadcastMessage`